### PR TITLE
Throttling update

### PR DESCRIPTION
**Pull request by michellefernando9**

### CODE REVIEW

**Review by marksou — DISMISSED** [review]
Looks good

**Review by carlosvillanua — DISMISSED** [review]
Thank you!

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/request-throttling.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Yes, you can. If you set `throttle_interval` and `throttle_retry_limit` values t
 
 3.  From the **Throttling** section, select the **Throttle interval** and the **Throttle retry limit** values.
     
-![Tyk API Gateway Request Throttling](/docs/img/dashboard/system-management/throttling_update.png)
+![Tyk API Gateway ottling](/docs/img/dashboard/system-management/throttling_update.png)
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/request-throttling.md]: typo

**Review by letzya — CHANGES_REQUESTED** [review]
there's still a typo

**Review by letzya — APPROVED** [review]

**Review by carlosvillanua — APPROVED** [review]
Thank you!

**Comment by letzya** [comment]
@marksou can you please merge?

**Pull request merged**